integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseUtils.java

```diff
@@ -107,8 +108,9 @@ public static FileOutStream createFile(FileSystem fileSystem, AuthPolicy authPol
    * @param uri the alluxio uri
    */
   public static void deleteFile(FileSystem fileSystem, AlluxioURI uri) {
+    DeletePOptions options = DeletePOptions.newBuilder().setUnchecked(true).build();
```

Comment:
Since this `options` is only used once as a local variable, I suggest we put it inline. e.g. 
`fileSystem.delete(uri, DeletePOptions.newBuilder....);`

Reply:
Adjusted

Reply:
does this mean deleteFile always deletes file in unchecked way? What if the user wants to do that in a checked way?

Reply:
Is it possible to have an option passed from the user?

Reply:
> does this mean deleteFile always deletes file in unchecked way?

Yes, if from alluxio-fuse

> Is it possible to have an option passed from the user?

It seems not feasible from the "rm" command for alluxio-fuse, the current AlluxioJniFuseFileSystem#unlink only has one parameter "String path". 

It's feasible to add a configuration parameter in "alluxio-site.properties", but I am not sure if it is needed, as the doc says "We recommend always using the -U option for the best performance and resource efficiency". And I don't any cases that deleting in unchecked way may have serious side effects.

@yyongycy please confirm if adding a parameter to the configuration file is needed. I will do it if yes.

Reply:
I am more concerned about the user's behavior. Some operations are not aligning with the real posix api definition, that may cause some misuse and unexpected behavior.  If the parameter is missing, it is better to add that parameter in the API.

@LuQQiu  What is your opinion?

Reply:
In another word, what is the exact requirement for this option?
1. rm the cache and UFS file?
2. rm the cache only?
3. rm the cache without the UFS?

Reply:
> what is the exact requirement for this option?

The background is that I created a lot of files under an alluxio-fuse path, but I cannot delete some of them with the "rm" command in a period because of "UFS dir not in sync. Sync UFS, or delete with unchecked flag." After checking the doc, I though making "unchecked" a default option of alluxio-fuse was ok.

Reply:
> If the parameter is missing, it is better to add that parameter in the API.

For the "rm" command, if executing with option "-f", it does not pass any flags to the system call "unlinkat". I am not sure if this is feasible.

Reply:
So I think the rm -f is feasible, "rm -f" means deleting a file whatever status it is, in cache or UFS or both.

Reply:
What I mean is, it seems that the option "-f" is handled by command "rm" and not passed to system call "unlinkat". While "AlluxioJniFuseFileSystem#unlink" is trigged by "unlinkat", so the option "-f" is unknowable to "AlluxioJniFuseFileSystem#unlink".  

Please correct me if I am wrong, thanks.

Reply:
I see what you mean.  
What if syncing the related dirs before deletion?
Would that be more elegant?

Reply:
> What if syncing the related dirs before deletion?

I think this is hard to achieve at client side with alluxio-fuse. One of my cases is that I created a lot of files and I want to delete some of them before they're synced to the under filesystem. For some users, they don't know the existence of alluxio and just do file operations to under the mount path. So it is hard to ask them do sync first or wait sync completed.